usecases/ast_eval/evaluate/eval_fuzzy_match.go

```diff
@@ -69,31 +64,25 @@ func (fuzzyMatcher FuzzyMatchAnyOf) Evaluate(ctx context.Context, arguments ast.
 
 	maxScore := 0
 	for _, rVal := range right {
-		maxScore = max(maxScore, f(left, pure_utils.CleanseString(rVal)))
+		maxScore = max(maxScore, f(left, rVal))
 		if maxScore == 100 {
 			break
 		}
 	}
 	return maxScore, nil
 }
 
-func getSimilarityAlgo(s string) (func(s1 string, s2 string, opts ...bool) int, error) {
-	var f func(s1 string, s2 string, opts ...bool) int
+func getSimilarityAlgo(s string) (func(s1 string, s2 string) int, error) {
+	var f func(s1 string, s2 string) int
+
 	switch s {
 	case "ratio":
-		f = func(s1 string, s2 string, opts ...bool) int { return fuzzy.Ratio(s1, s2) }
-	case "partial_ratio":
-		f = func(s1 string, s2 string, opts ...bool) int { return fuzzy.PartialRatio(s1, s2) }
-	case "token_sort_ratio":
-		f = fuzzy.TokenSortRatio
+		return pure_utils.DirectSimilarity, nil
 	case "token_set_ratio":
```

Comment:
IMO, we should add acomment to explain to future "us" (or devs) why this case exists